pipeline/data_merging/bayesdel/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    parallel
+    #&& rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /opt/victor
+
+WORKDIR /opt/victor
+
+RUN curl -L -o VICTOR_linux.tgz https://www.dropbox.com/s/rkfu47k8o4mh5wg/VICTOR_linux.tgz && tar zxf VICTOR_linux.tgz && rm -rf VICTOR_linux.tgz
+
+# TODO split up and do it manually
+
+# installing htslib (includes tabix and bgzip)
+
+WORKDIR /opt
+
+ARG HTSLIB_VERSION=1.10.2
+# TODO squash it together
```

Comment:
still a TODO? not clear to me what this is referring to, but prob best to either do it now or remove the todo